src/test/java/org/jitsi/meet/test/base/FailureListener.java

```diff
@@ -114,7 +115,15 @@ public void onStart(ITestContext iTestContext)
 
     @Override
     public void onFinish(ITestContext iTestContext)
-    {}
+    {
+        // move all failed configurations to the failed tests set
```

Comment:
so is it that failed configurations do not appear at all in the report ?

Reply:
Nope, its that we will see that test also as failed.